### PR TITLE
{cae}[intel/2015b] FIAT 1.6.0 w/ Python 2.7.11 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FIAT/FIAT-1.6.0-intel-2015b-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+pyver = '2.7.11'
+versionsuffix = '-Python-%s' % pyver
+
+dependencies = [
+    ('Python', pyver),
+    ('sympy', '0.7.6.1', versionsuffix),
+]
+
+options = {'modulename': name}
```
Comment (wpoely86) [inline easybuild/easyconfigs/f/FIAT/FIAT-1.6.0-intel-2015b-Python-2.7.11.eb]: isn't this the default?

File: easybuild/easyconfigs/s/sympy/sympy-0.7.6.1-intel-2015b-Python-2.7.11.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+python = 'Python'
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[:-1])
+
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [(python, pyver)]
+
+runtest = 'python setup.py test'
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/sympy/sympy-0.7.6.1-intel-2015b-Python-2.7.11.eb]: default?

File: easybuild/easyconfigs/f/FIAT/FIAT-1.6.0-intel-2015b-Python-2.7.11.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/f/FIAT/FIAT-1.6.0-intel-2015b-Python-2.7.11.eb]: no, the default is lowercase (`fiat` vs `FIAT`), which is far more common

File: easybuild/easyconfigs/s/sympy/sympy-0.7.6.1-intel-2015b-Python-2.7.11.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/s/sympy/sympy-0.7.6.1-intel-2015b-Python-2.7.11.eb]: no, there is no default currently implemented by `PythonPackage`, it just provides support to specify a test command and set things up so the tests can run...

not sure if this is common enough to be the default

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b5e3d4e52fbd76ec7476 for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**